### ISSUE TITLE
Graphic Glitches : Castlevania III - Dracula's Curse (USA)

### ISSUE THREAD

**Issue by rgavril** [MiSTer-devel/NES_MiSTer#193]
In the current version (NES_20200526.rbf), the title screen and the first level has some graphical glitches that make the game unusable. 

They were not were not present in the previous version (NES_20200522.rbf).

**Comment by SegaSnatcher** [comment]
I'm not getting any graphical glitches.

**Comment by rgavril** [comment]
I've downloaded a new rom and it works fine :/

It seams that the one that I had was not the official no-intro set (it had a sha1 of 091a07dda25099e73f50878316c82fb501d3c965). What is strange is that is used to work in previous versions and the this same rom works fine with various NES emulators. 

From my point of view the issues can be closed but I've uploaded some images maybe it helps you catch an hidden bug that was visible using that rom.

I currently have MISTer v200601 / NES v200526, everything was updated with the script.
![IMG_4838](https://user-images.githubusercontent.com/5073422/83519262-54c64480-a4e4-11ea-9ceb-ed1963b6ac75.jpeg)
![IMG_4839](https://user-images.githubusercontent.com/5073422/83519270-5859cb80-a4e4-11ea-9342-e46f545a096f.jpeg)
